README.md

```diff
@@ -48,11 +48,18 @@ These instructions will get you a copy of the project up and running on your loc
 1. Open terminal on Mac or command line on Windows. We like to use [VSCode's integrated terminal](https://code.visualstudio.com/docs/editor/integrated-terminal), as it works for both Mac and PC.
 1. Clone repo. `git clone {paste URL you copied}`
 1. Navigate into repo. `cd hack-la-203`
-1. Create a folder using your group name. All of your work should be contained inside this folder to reduce the likelihood of merge conflicts when you submit your work later. 
-1. Copy the [final_submission_template.md](./final_submission_template.md) into the folder you created.
+1. Rename the `src_rename_me` folder with your group name. All of your work should be contained inside this folder to reduce the likelihood of merge conflicts when you submit your work later. 
+1. Download your events data from the Canvas quiz and place under the `data` folder, and rename the file as `events.csv`. Anything that is contained in this folder is gitignored.
```

Comment:
can the data folder be in the src_rename_me template with a .gitignore file within.

Reply:
I moved the `data` folder to inside `read_me` now.